src/frontend/scss/settings/_extra_variables.scss

```diff
@@ -0,0 +1,167 @@
+// One pixel shortcut
```

Comment:
I'll not move those variables into a dedicated file 🤷‍♂️

Reply:
Ok, no problem. Now it would be much better.